trunner/target/armv7a7.py

```diff
@@ -89,15 +69,20 @@ def build_test(self, test: TestOptions) -> Callable[[], Optional[TestResult]]:
 
 class IMX6ULLEvkTarget(ARMv7A7Target):
     # IMX6ULL with system jffs2 use nor0 as space to hold data
-    image = PloImageProperty(file="phoenix.disk", source="usb0", memory_bank="nor0")
-    name = "armv7a7-imx6ull-evk"
-    cleanmarkers_spec = PloJffs2CleanmarkerSpec(
-        start_block=0x10,
-        number_of_blocks=0x1F0,
-        block_size=0x10000,
-        cleanmarker_size=0x10,
+    image = PloJffsImageProperty(
+        file="phoenix.disk",
+        source="usb0",
+        memory_bank="nor0",
+        flash_device_id="2.0",
+        cleanmarkers_args=PloJffs2CleanmarkerSpec(
+            start_block=0x10,
+            number_of_blocks=0x1F0,
```

Comment:
We rather use lowercase letters in hex.

```suggestion
            number_of_blocks=0x1f0,
```